edx_proctoring/tests/test_views.py

```diff
@@ -736,6 +760,79 @@ def test_ineligible_for_onboarding_exam(self):
         message = 'There is no onboarding exam accessible to this user.'
         self.assertEqual(response_data['detail'], message)
 
+    def test_multiple_hidden_onboarding_exams(self):
```

Comment:
Can we have a test case where none of the exams within the course content is visible to the learner? The test case is to mimic the Masters learner in GT course case. The Masters learner should not see any onboarding exams.

Reply:
Sounds like a good idea to test for no exams. This won't entirely handle the GT case however since the exam sections are still accessible to the learner.